internal/cmd/options.go

```diff
@@ -15,3 +20,22 @@ func (opts *SignerFlagOpts) ApplyFlags(fs *pflag.FlagSet) {
 	SetPflagKey(fs, &opts.Key)
 	SetPflagSignatureFormat(fs, &opts.SignatureFormat)
 }
+
+// LoggingFlagOpts option struct.
+type LoggingFlagOpts struct {
+	Debug bool
+}
+
+// ApplyFlags applies flags to a command flag set.
+func (opts *LoggingFlagOpts) ApplyFlags(fs *pflag.FlagSet) {
+	fs.BoolVarP(&opts.Debug, "debug", "d", false, "debug mode")
+}
+
+// SetLoggerLevel sets up the logger based on common options.
+func (opts *LoggingFlagOpts) SetLoggerLevel() (context.Context, log.Logger) {
+	var logLevel logrus.Level
```

Comment:
In logrus, although logrus.New() defaults to info level, logrus.Level's default is panic.